types/bumblebee-hotword/bumblebee-hotword-tests.ts

```diff
@@ -0,0 +1,50 @@
+import Bumblebee , { VoiceProcessor , SpectrumAnalyser } from 'bumblebee-hotword';
```

Comment:
Personally, I like to match the import style in the sample code in the project readme: `const BumbleBee = require('bumblebee-hotword');`.

Reply:
I can make a changes to that.

Reply:
but seems tslint will complain about that.